File: src/frontend/core/components/dto/AuthDTO.ts
export type SignInSuccessDTO = {
    userId: number,
    accessToken: string,
}

export type SignInDTO = {
    userName: string,
    password: string,
}

export type ChangePwDTO = {
    signIn: SignInDTO
    newPw: string
}

export type SignInAdminDTO = {
    userName: string,
    password1: string,
    password2: string,
}

export type ChangePwAdminDTO = {    
    signIn: SignInAdminDTO,
    newPw: string,
}

File: src/backendOld/core/DTO/AuthDTO.cs
namespace SnippetVault {


public class SignInSuccessDTO {
    public int userId {get; set; }
}

public class SignInDTO {
    public string userName {get; set;}
    public string password {get; set;}
}

public class ChangePwDTO {
    public SignInDTO signIn {get; set;}
    public string newPw {get; set;}
}

public class SignInAdminDTO {
    public string userName {get; set; }
    public string password1 {get; set; }
    public string password2 {get; set; }
}

public class ChangePwAdminDTO {
    public SignInAdminDTO signIn {get; set;}
    public string newPw {get; set;}
}

}
